packages/frontend/editor-ui/src/Interface.ts

```diff
@@ -136,6 +136,8 @@ declare global {
 	}
 }
 
+export type Prettify<T> = { [K in keyof T]: T[K] } & {};
```

Comment:
I like the idea of this, but it's not used in the codebase. Maybe you can delete it.

It would be nice, though, to have something like this as an IDE plugin for better intersection type discovery.

Reply:
Agree, I was using it when I had resource types hard-coded here so it's just a leftover at this point.
Happy to delete but figured might be useful moving forward